app/controllers/favorites_controller.rb

```diff
@@ -1,4 +1,4 @@
-class FavoriteController < ApplicationController
+class FavoritesController < ApplicationController
```

Comment:
I'm glad these are working, we'll likely find a few more when we get this merged in with story 16

Reply:
yes, i did find a few more last night. they're fixed in the us16 branch